conform-pr/main_test.go

```diff
@@ -88,10 +101,12 @@ func TestGetPR(t *testing.T) {
 		})
 
 		action := githubactions.New(githubactions.WithGetenv(getEnv))
-		pr, err := getPR(action)
+		pr, err := getPR(action, client)
 		assert.NoError(t, err)
 		assert.Equal(t, "Add Docker badge", pr.title)
 		assert.Equal(t, "This PR is a sample PR \n\nrepresenting a body that ends with a dot.", pr.body)
+		//	assert.Equal(t, "PR_kwDOGmfjh84yFEcA", pr.nodeID)
+		//	assert.Equal(t, []string{"Sprint 1", "Sprint 2", "Sprint 3"}, pr.sprints)
```

Comment:
Do we need those?

Reply:
removed (no, we don't need them, we test it in `internal/graphql`)